README.md

```diff
@@ -8,6 +8,8 @@
 Adds compression utils to [the Fastify `reply` object](https://www.fastify.io/docs/master/Reply/) and a hook to decompress requests payloads.
 Supports `gzip`, `deflate`, and `brotli`.
 
+> **Important note:** since `fastify-compress` version 4.x payloads that are compressed using the `zip` algorithm are not automatically uncompressed anymore as it is out of the scope of this plugin.
```

Comment:
Explain why

Reply:
Hmmm maybe something like this will be more informative. WDYT ?
I am not sure about the english formulation though.

> **Important note:** since `fastify-compress` version 4.x payloads that are compressed using the `zip` algorithm are not automatically uncompressed anymore. `fastify-compress` main feature is to provide response compression mechanism to your server, however the `zip` format does not appear in the [IANA maintained Table of Content Encodings](https://www.iana.org/assignments/http-parameters/http-parameters.xml#content-coding) and thus such behavior was out of the scope of this plugin.

The big idea here is that when you stream a zip archive from your server you want that the end user receives the zip archive and not the uncompressed content of this same archive and this plugin should stick to the algorithms allowed by the spec and should not "alter" zip archives. :d